### PR TITLE
Linter CLI: Preserve spacing in new rule hints

### LINKED ISSUE #2021: Linter CLI: Missing space after rule name in `New rules available` hint
Missing whitespace after rule name in the `New rules available` output
```
 New rules available:
  Your .herb.yml version is 0.10.3. 11 new rules are disabled to ease upgrades:
  from Herb config: /home/runner/work/rubyevents/rubyevents/.herb.yml

  actionview-no-helper-shadowing(introduced in next release)
  actionview-no-redundant-local-assigns(introduced in next release)
  actionview-no-unnecessary-html-safe(introduced in next release)
  actionview-no-unused-strict-locals(introduced in next release)
  actionview-prefer-collection-render(introduced in next release)
  erb-no-shadowed-block-argument(introduced in next release)
  erb-no-unused-block-argument(introduced in next release)
  erb-prefer-do-end-blocks(introduced in next release)
  erb-prefer-each-over-map(introduced in next release)
  html-no-nested-forms(introduced in next release)
  turbo-permanent-no-misleading-value(introduced in next release)
```

### PR DESCRIPTION
**Pull request by joeeei11**
## Summary

- keep the separating whitespace inside the gray ANSI sequence for version labels
- add a regression test for the hyperlink-to-color escape boundary

## Testing

- `yarn workspace @herb-tools/linter test test/cli.test.ts`

Closes #2021

### CODE REVIEW

File: javascript/packages/linter/test/summary-reporter.test.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { SummaryReporter } from "../src/cli/summary-reporter.js"
+
+import type { SummaryData } from "../src/cli/summary-reporter.js"
+
+describe("SummaryReporter", () => {
+  afterEach(() => {
+    vi.restoreAllMocks()
+  })
+
+  test("keeps the space before a version label inside its color sequence", () => {
```
Comment (marcoroth) [inline javascript/packages/linter/test/summary-reporter.test.ts]: @joeeei11 would you mind moving this to the existing `cli.test.ts` and using `toMatchSnapshot` on the output.

**Comment by joeeei11** [comment]
I'm glad to hear back from you. As this is my first PR, I look forward to more opportunities to collaborate with you in the future.



---Original---
From: "Marco ***@***.***&gt;
Date: Tue, Aug 11, 2026 04:20 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [marcoroth/herb] Linter CLI: Preserve spacing in new rule hints (PR #2022)




 Merged #2022 into main.
